docs/man/man1/kubectl-exec.1

```diff
@@ -19,7 +19,7 @@ Execute a command in a container.
 .SH OPTIONS
 .PP
 \fB\-c\fP, \fB\-\-container\fP=""
-    Container name. If omitted, the first container in the pod will be chosen
+    Container name. If omitted, the first app container in the pod will be chosen
```

Comment:
```
the first app container in the pod will be chosen
```
Here I just want to use a more accurate help info, we will use the first non init container. Guess we couldchange this to `user container`.